tokio/src/sync/semaphore.rs

```diff
@@ -210,6 +210,55 @@ use std::sync::Arc;
 /// }
 /// ```
 ///
+/// ## Ensure Tests with Mutual Dependencies Run Sequentially
+///
+/// By default, Rust runs tests concurrently. However, in some scenarios, concurrent execution might lead to test interference.
+/// For example, multiple tests might communicate with the same database, necessitating sequential execution to prevent data conflicts.
```

Comment:
I think the language here is somewhat complicated. I'd like to simplify it. For example:

> In some cases, running two tests in parallel may lead to problems. For example, this can happen when tests use the same database.

Reply:
done.